solutions/swb-reference/integration-tests/tests/isolated/projects/listDataSetsForProject.test.ts

```diff
@@ -0,0 +1,131 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+import ClientSession from '../../../support/clientSession';
+import { PaabHelper } from '../../../support/complex/paabHelper';
+import HttpError from '../../../support/utils/HttpError';
+import { checkHttpError } from '../../../support/utils/utilities';
+
+describe('list datasets for project tests', () => {
+  let paabHelper: PaabHelper;
+  let itAdminSession: ClientSession;
+  let researcher1Session: ClientSession;
+  let pa1Session: ClientSession;
+  let pa2Session: ClientSession;
+  let project1Id: string;
+
+  beforeEach(() => {
+    expect.hasAssertions();
+  });
+
+  beforeAll(async () => {
+    paabHelper = new PaabHelper();
+    const paabResources = await paabHelper.createResources();
+    itAdminSession = paabResources.adminSession;
+    researcher1Session = paabResources.rs1Session;
+    pa1Session = paabResources.pa1Session;
+    pa2Session = paabResources.pa2Session;
+    project1Id = paabResources.project1Id;
+  });
+
+  afterAll(async () => {
+    await paabHelper.cleanup();
+  });
+
+  describe('negative tests', () => {
+    test('IT Admin cannot view any datasets, so the call will return an empty list', async () => {
+      const { data } = await itAdminSession.resources.projects.project(project1Id).dataSets().get();
+
+      expect(data.data).toStrictEqual([]);
+    });
+
+    test('Project Admin from project 2 cannot list datasets for project 1', async () => {
+      try {
+        await pa2Session.resources.projects.project(project1Id).dataSets().get();
```

Comment:
Possible negative case: User belongs to a group with access to dataset 1 through project 1 but is trying to access dataset 1 through project 2?

Reply:
Will handle this test in another PR.